deps/rabbitmq_peer_discovery_k8s/src/rabbit_peer_discovery_k8s.erl

```diff
@@ -95,16 +96,12 @@ lock(Node) ->
       Error
   end.
 
--spec unlock({ResourceId :: string(), LockRequesterId :: node()}) -> ok | {error, Reason :: string()}.
+-spec unlock({{ResourceId :: string(), LockRequesterId :: node()}, Nodes :: [node()]}) ->
+    ok | {error, Reason :: string()}.
```

Comment:
Can we now omit the `{error, Reason :: string()}` part in the function spec?

Running `make dialyze` within folder `deps/rabbitmq_peer_discovery_k8s` will return:
```
rabbitmq_peer_discovery_k8s.erl:51:2: Invalid type specification for function rabbitmq_peer_discovery_k8s:unlock/1. The success typing is
          ({{string(), atom()}, [atom()]}) -> 'ok'
```
So, we'd also need to modify the function spec in that other file.

Reply:
@ansd this has already been merged, so if the spec were to be adjusted, it will have to be a new PR.

Reply:
ah right. I forgot about that. Also, I thought the default make target would also run the spec analyzer, but that isn't the case.